templates/emails/activation_email.txt

```diff
@@ -0,0 +1,38 @@
+<%namespace file="../main.html" import="stanford_theme_enabled" />
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+
+
+% if is_secure:
+  <a href="https://${ site }/activate/${ key }">
+% else:
+  <a href="http://${ site }/activate/${ key }">
+% endif
+
+    <img src="http://courses.gymna.si/static/themes/gymnasium/images/gym-reg-activate.png" alt="click to activate">
```

Comment:
I don't think that Open edX supports HTML email templates for registration/activation yet. I'm pretty sure there was a PR a while ago to add this feature to edx-platform, but I can't find it at the moment and I don't know the status of it.

Were you able to test this out?